me.simakis.dropboxignore.yml

```diff
@@ -0,0 +1,19 @@
+app-id: me.simakis.dropboxignore
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: dropboxignore.sh
+
+modules:
+ - name: dropboxingore
+   buildsystem: simple
+   build-commands:
+     - install -D bin/dropboxignore.sh /app/bin/dropboxignore.sh
+     - install -Dm644 icons/64.png /app/share/app-info/icons/flatpak/64x64/me.simakis.dropboxigmore
+     - install -Dm644 icons/128.png /app/share/app-info/icons/flatpak/128x128/me.simakis.dropboxigmore
+     - install -Dm644 icons/256.png /app/share/app-info/icons/flatpak/256x256/me.simakis.dropboxigmore
+     - install -Dm644 icons/512.png /app/share/app-info/icons/flatpak/512x512/me.simakis.dropboxigmore
```

Comment:
you NEVER install in app-info. Also when installing the icon file you keep its extension

Neither the metainfo nor the desktop files are installed.

Reply:
Almost certain that I fixed the issues.